api/controllers/game/concede.js

```diff
@@ -5,14 +5,16 @@ module.exports = async function (req, res) {
     await Game.updateOne(req.session.game).set({ result: winner });
     const game = await gameService.populateGame({ gameId: req.session.game });
     await gameService.clearGame({ userId: req.session.usr });
+    let currentMatch;
     if (game.isRanked) {
-      await sails.helpers.addGameToMatch(game);
+      currentMatch = await sails.helpers.addGameToMatch(game);
     }
```

Comment:
Something like this might work better:
```
const currentMatch = game.isRanked ? await sails.helpers.addGameToMatch(game) : null;
```
That way the key always comes through but is set according to the context.

Reply:
I like this as well. Keeps a consistent payload shape